NiimPrintX/nimmy/printer.py

```diff
@@ -265,6 +290,10 @@ async def set_label_density(self, n):
     async def start_print(self):
         packet = await self.send_command(RequestCodeEnum.START_PRINT, b"\x01")
         return bool(packet.data[0])
+    
+    async def start_printV2(self):
+        packet = await self.send_command(RequestCodeEnum.START_PRINT, b"\x00\x01\x00\x00\x00\x00\x00")
```

Comment:
First two bytes are indicating total page count of print. Here you send only one page. You should use variable. Otherwise set_dimensionV2 will not work for >1 pages.

Reply:
You are right, I put that in for testing and then did not remove it.
Will add it later, thank you for the feedback.